source/exe/hot_restart.h

```diff
@@ -201,5 +205,39 @@ class HotRestartImpl : public HotRestart,
   bool parent_terminated_{};
 };
 
+#else  // __linux__
+
+/**
+ * No-op implementation of HotRestart for everybody else.
```

Comment:
We (@ Google) would find this useful even on Linux. @mrice32. Maybe there is some way to condition this at build time.